### ISSUE TITLE
First run of esy-bash has extraneous output

### ISSUE THREAD

**Issue by bryphe** [bryphe/esy-bash#29]
This causes failures like this the [with this build](https://ci.appveyor.com/project/esy/esy-opam-override/builds/19700662):
```
error: sha1 checksum mismatch: expected 59ebdbbaedcda123fc7ed8fb2b302b7d819e9a46 but got Copying
  fetching archive:https://registry.npmjs.org/@esy-ocaml/substs/-/substs-0.0.1.tgz#sha1:59ebdbbaedcda123fc7ed8fb2b302b7d819e9a46
esy: exiting due to errors above
```
(The extra output broke the result parsing!)

**Comment by prometheansacrifice** [comment]
@bryphe Looks like it has to do with how stdout is handled, isn't it?

**Comment by bryphe** [comment]
The output that breaks us is stuff like:
```
Copying skeleton files.
These files are for the users to personalise their cygwin experience.
They will never be overwritten nor automatically updated.
'./.bashrc' -> '/home/appveyor//.bashrc'
'./.bash_profile' -> '/home/appveyor//.bash_profile'
'./.inputrc' -> '/home/appveyor//.inputrc'
'./.profile' -> '/home/appveyor//.profile'
```

**Comment by prometheansacrifice** [comment]
Unfortunately, I couldn't repro this locally on my machine.

**Comment by bryphe** [comment]
Thanks for testing it out @prometheansacrifice !

I think the issue is that, if you don't have a `.bashrc`/`.bash_profile` etc already, the first run of bash will create - this breaks the output we expect! So it might not repro locally if you already have those in your home directory.

I think there are a couple things we need to do here:
- __Explicitly set the home directory__ - we can set this in the `/etc/nsswitch.conf`. We can set the home directory by using `db_home`. We could set it like: `db_home: /usr/esy`. We could include this modified `/etc/nsswitch.conf` in our bundle.
- __Seed the home directory with the requisite scripts__ - we could just run `bash.exe` prior to packing to get the `.bashrc`/`.bash_profile`/etc generated

**Comment by bryphe** [comment]
Fixed by #35 - thank you @prometheansacrifice ! 💯
